internal/pkg/service/appsproxy/proxy/server.go

```diff
@@ -86,8 +135,10 @@ func NewHandler(d dependencies.ServiceScope) http.Handler {
 		middleware.RequestInfo(),
 		middleware.Filter(middlewareCfg),
 		middleware.Logger(d.Logger()),
+		// Mandatory middleware when used in combination with newTracerProviderWrapper
+		middleware.Request(),
```

Comment:
There is already the `RequestInfo` middleware, I would move the code there, and not create a new middleware.

Reply:
Fixed.